packages/web/src/js/element/lines.ts

```diff
@@ -369,17 +369,7 @@ export const renbanInfo: ElementInfo = {
     getWarnings(value: schema.LineElement['value'], _grid: Grid, digits: IdxMap<Geometry.CELL, number>, warnings: IdxBitset<Geometry.CELL>): void {
         outer:
         for (const cells of Object.values(value || {})) {
-            const cellsArr = arrayObj2array(cells);
-
-            const uniqueCellsArr = cellsArr.filter(
-                (n,i, arr) => {
-                    return arr.findIndex(
-                        t => {
-                            if(n === t)
-                                return true;
-                            return n[0] === t[0] && n[1] === t[1];
-                        }) === i;
-                })
+            const uniqueCellsArr = [...new Set(arrayObj2array(cells))];
 
```

Comment:
Nice, this appears to fix a bug with the conflict highlighter not working for renbans.

![chrome_joI9NzoNKu](https://user-images.githubusercontent.com/8262173/211363620-eb3f8004-97c6-46d9-9469-87f3f9d32486.png)

Reply:
Yeah. I introduced that bug.
I didn't notice that the 2 areas where we handle renbans being a loop had arrays with different types in them